### ISSUE TITLE
Cannot read property 'duplicatePackages' of undefined

### ISSUE THREAD

**Issue by yurii-github** [relative-ci/bundle-stats#434]
file **bundle-stats.html** produces error
`Uncaught TypeError: Cannot read property 'duplicatePackages' of undefined`

![image](https://user-images.githubusercontent.com/1162656/67640352-d9276800-f902-11e9-9f83-0ce1b889cb4f.png)

```
  "devDependencies": {
    "@webpack-cli/info": "^0.1.6",
    "bundle-stats": "^1.12.0",
    "clean-webpack-plugin": "^3.0.0",
    "css-loader": "^3.2.0",
    "file-loader": "^4.2.0",
    "html-webpack-plugin": "^3.2.0",
    "style-loader": "^1.0.0",
    "webpack": "^4.41.2",
    "webpack-cli": "^3.3.9",
    "webpack-dev-server": "^3.9.0",
    "webpack-manifest-plugin": "^2.2.0"
  },
  "dependencies": {
    "lodash": "^4.17.15"
  }
```

```
yarn list | grep bundle-stats
├─ @bundle-stats/html-templates@1.12.0
├─ @bundle-stats/utils@1.12.0
├─ bundle-stats@1.12.0
│  ├─ @bundle-stats/html-templates@^1.12.0
│  ├─ @bundle-stats/utils@^1.12.0

```

```
 yarn list | grep webpack
├─ @types/webpack-sources@0.1.5
├─ @types/webpack@4.39.5
│  ├─ @types/webpack-sources@*
├─ @webpack-cli/info@0.1.6
├─ clean-webpack-plugin@3.0.0
│  ├─ @types/webpack@^4.4.31
├─ html-webpack-plugin@3.2.0
├─ terser-webpack-plugin@1.4.1
│  ├─ webpack-sources@^1.4.0
├─ webpack-cli@3.3.9
├─ webpack-dev-middleware@3.7.2
│  └─ webpack-log@^2.0.0
├─ webpack-dev-server@3.9.0
│  ├─ webpack-dev-middleware@^3.7.2
│  ├─ webpack-log@^2.0.0
├─ webpack-log@2.0.0
├─ webpack-manifest-plugin@2.2.0
├─ webpack-sources@1.4.3
├─ webpack@4.41.2
│  ├─ terser-webpack-plugin@^1.4.1
│  └─ webpack-sources@^1.4.1

```

**Comment by vio** [comment]
Thanks for reporting it @yurii-github, I dropped the ball on this one!

I fixed it and released a patch version (`1.12.1`), can you please install the latest and try again?

**Comment by yurii-github** [comment]
```
 yarn list | grep bundle-stats
├─ @bundle-stats/html-templates@1.12.1
├─ @bundle-stats/utils@1.12.0
├─ bundle-stats@1.12.1
│  ├─ @bundle-stats/html-templates@^1.12.1
│  ├─ @bundle-stats/utils@^1.12.0
```
thanks for your fix and this project! I do confirm - the error has gone.
